app/assets/stylesheets/landing_page.scss

```diff
@@ -0,0 +1,38 @@
+@import 'bootstrap/dist/css/bootstrap.min';
+
+body {
+  position: fixed;
+  background-color: pink;
+  background-image: url('background1.jpg');
+  height: 100%;
+  width: 100%;
+  color: black;
+}
+
+.container {
+  background-color: darkgrey;
+  width: 1000px;
+  height: 500px;
+  top: 50%;
+  left: 50%;
+  margin-top: 90px;
+  margin-left: 130px;
+}
+
+.login-info {
+  background-color: white;
+  float: left;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 50%;
+}
+
+.main-header {
+  top: 50px;
+  left: 50px;
+}
+
+.navbar-nav {
```

Comment:
Single declarations should be a single line.

Reply:
interesting, I haven't used it that way, but we can configure the linter that way (when we finally have one :D)